test/e2e/apimachinery/webhook.go

```diff
@@ -262,12 +277,35 @@ func testWebhook(f *framework.Framework) {
 	// TODO: Test if webhook can detect pod with non-compliant metadata.
 	// Currently metadata is lost because webhook uses the external version of
 	// the objects, and the apiserver sends the internal objects.
+
+	By("create a namespace that bypass the webhook")
```

Comment:
Is this allowed in the test framework? Do you need to modify an existing namespace instead?